File: 백준/Bronze/15059. Hard choice/Hard choice.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const [ca, ba, pa] = input[0].split(' ').map(Number);
const [cr, br, pr] = input[1].split(' ').map(Number);

const c = cr - ca >= 0 ? cr - ca : 0;
const b = br - ba >= 0 ? br - ba : 0;
const p = pr - pa >= 0 ? pr - pa : 0;

console.log(c + b + p);

File: 백준/Bronze/15059. Hard choice/Hard choice.py
ca, ba, pa = map(int, input().split())
cr, br, pr = map(int, input().split())

c = cr - ca if cr - ca >= 0 else 0
b = br - ba if br - ba >= 0 else 0
p = pr - pa if pr - pa >= 0 else 0

print(c + b + p)